sonar-ruby-plugin/src/main/resources/org/sonar/l10n/ruby/rules/ruby/S1066.html

```diff
@@ -0,0 +1,21 @@
+<p>Merging collapsible <code>if</code> statements increases the code's readability.</p>
```

Comment:
You don't need to put description section to lang subtask if it's the same, it's a separate section and it will be "inherited" during generation

Reply:
Yep, it was a copy-paste mistake. Removed description section from subtask